tests/Sniffs/Commenting/data/deprecatedDocRequiresDescription.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @deprecated
+ */
+class Whatever
+{
+	/**
+	 * @deprecated
+	 */
+	public const DEPRECATED_WITHOUT = 'deprecated';
```

Comment:
Please use tabs in the whole file.

Reply:
@kukulich you might want to move this check (tabs) to the CI level